VocableUITests/Tests/MainScreenPaginationTests.swift

```diff
@@ -0,0 +1,136 @@
+//
+//  MainScreenPaginationTests.swift
+//  VocableUITests
+//
+//  Created by Rudy Salas on 4/8/22.
+//  Copyright © 2022 WillowTree. All rights reserved.
+//
+
+import XCTest
+
+class MainScreenPaginationTests: CustomPhraseBaseTest {
+    
+    // In order to ensure this test passes on both an iPhone and iPad device, we will add 16 phrase (2 pages on an iPad)
+    // then remove 8 of them so that the pages reduce from 2 to 1 page.
+    func testDeletingPhrasesAdjustsPagination() {
+        listOfPhrases.forEach{ phrase in
```

Comment:
```suggestion
        listOfPhrases.forEach { phrase in
```

This would technically be a swiftlint warning without the spacing after the `forEach` declaration